src/span-data.ts

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import * as crypto from 'crypto';
```

Comment:
I'm wondering if the use of cryptographically-secure random bytes is necessary here. Would we be able to use Math.random() instead?

Reply:
Sure, I switched this to use `Math.random()`.

My thinking with the crypto random was that if two Node processes run on a machine with a common clock (e.g. containers on the same VM), perhaps their Math.random would be seeded to the same number and they would produce colliding span IDs. But I think my concern is probably unwarranted and random number generators are probably smarter these days.